sensors/video/sensorhub-driver-v4l/build.gradle

```diff
@@ -27,6 +27,11 @@ osgi {
   manifest {
     attributes('Bundle-Vendor': 'Sensia Software LLC')
     attributes('Bundle-Activator': 'org.sensorhub.impl.sensor.v4l.Activator')
+    attributes('-exportcontents': 'org.sensorhub.impl.sensor.v4l, org.sensorhub.impl.sensor.v4l')
+    attributes('Import-Package': '''
+      !javax.imageio.*, !java.*, !javax.*,
+      org.osgi.service.repository;resolution:=optional,
```

Comment:
Don't think we need to import "org.osgi.service.repository;resolution:=optional"
Also isn't `!javax.imageio.*` redundant with `!javax.*`?

Reply:
Ok for building and installing: 5ddd4211df4bd9220d309082033b101d7c922127

Reply:
remove redundant exclude: ed5ac0de56877635278a844c33f32450f629cffd